### PR TITLE
Create gas-fee-on-flow-evm.md

### PR DESCRIPTION
**Pull request by KshitijChaudhary666**
Developer doc on EVM gas fee

### CODE REVIEW

**Comment by gregsantos** [comment]
Looks good. Links need updating for deployment it seems

File: docs/evm/Build/Guides/gas-fee-on-flow-evm.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# **How gas fees work on Flow EVM?**
+
+### ***Revisiting Transaction Fee on Flow***
+
+Transaction Fee on Flow is broken down into [three components](https://developers.flow.com/learn/concepts/variable-transaction-fees) - ‘*Inclusion Fee*’ that accounts for the resources required to process a transaction due to the transaction’s core properties (byte size, number of signatures); ‘Execution Fee’ that accounts for the operational cost of running the transaction script, processing the results, sending results for verification, generating verification receipt, etc; and a *Surge* factor as a multiplicative factor to dynamically account for network pressure and market conditions.
+
+***Transaction fee = {inclusion fee + (execution effort * unit cost)} x surge***
```
Comment (vishalchangrani) [inline docs/evm/Build/Guides/gas-fee-on-flow-evm.md:7]: Line 7 is being flagged as an [error](https://github.com/onflow/docs/actions/runs/8010878669/job/21882877890?pr=588#step:7:59) by Vercel

Comment (vishalchangrani) [reply docs/evm/Build/Guides/gas-fee-on-flow-evm.md:7]: https://github.com/micromark/micromark-extension-mdx-expression/tree/main/packages/micromark-extension-mdx-expression#could-not-parse-expression-with-acorn

Comment (vishalchangrani) [reply docs/evm/Build/Guides/gas-fee-on-flow-evm.md:7]: I think the open braces `{` is causing it to evaluate the text as an expression

Comment (KshitijChaudhary666) [reply docs/evm/Build/Guides/gas-fee-on-flow-evm.md]: Changed, thank you!

Comment (vishalchangrani) [reply docs/evm/Build/Guides/gas-fee-on-flow-evm.md]: 🤞

**Review by vishalchangrani — APPROVED** [review]

**Pull request merged**